packages/documentation/library/pages/about-the-design-system/index.md

```diff
@@ -4,6 +4,7 @@ description: ''
 tags: public
 pageClass: ''
 template: default
+index: 6
```

Comment:
It would be better if we could remove the dependencies on adding stuff to the markdown files - Ideally we'd actually limit the markdown frontmatter to `title`, `description` and `template`. If this absolutely has to be here then I suggest you at least name it something that's non-developer friendly like `navPosition` or something a bit more descriptive.

Reply:
Without some form of draggable UI that lets you order pages (like in a CMS) i'm not sure how else you would define this? 

Open to suggestions.

The index is relative to pages within that directory.

Reply:
An alternative might be to add a numerical index to the start of the filename:

```
  ├── directory
  │   └── 1__example-page.md
  ├── another-directory
  │   ├── 1__example-page.md
  │   └── 2__another-example-page.md
```

Reply:
I like the meta tag approach. Don't like depending on file names or folders to get the order right. Limits us when we want to pull in component documentation from the component folder